### PR TITLE
chore: mobile testing

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [x] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes # 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -64,6 +64,7 @@ jobs:
             demo,
             kit,
           ]
+        config: [cypress.config.ts, cypress-mobile.config.ts]
```
Comment (splincode) [inline .github/workflows/e2e.yml:67]: image

`name: ${{ matrix.project }} / ${{ matrix.type }}`

Comment (splincode) [inline .github/workflows/e2e.yml:67]:
```
type: [desktop, mobile]
```

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -86,7 +87,10 @@ jobs:
         run: npx nx serve-compiled --path ${{ env.LOCAL_REF_DEMO }} --port ${{ env.NG_SERVER_PORT }}
 
       - name: Run screenshot tests on ${{ env.LOCAL_REF_DEMO }}
-        run: npx nx e2e-ci --key localhost --folder ${{ matrix.project }}
+        run: npx nx e2e-ci --key localhost --folder ${{ matrix.project }} --config ${{ matrix.config }}
```
Comment (splincode) [inline .github/workflows/e2e.yml:90]: `run: npx nx e2e-ci --key localhost --folder ${{ matrix.project }} --config cypress.config.${{ matrix.type }}.ts`

File: projects/demo-integrations/cypress.config.ts [hunk 3]
```diff
@@ -44,7 +44,7 @@ export const TUI_CYPRESS_CONFIG: Cypress.ConfigOptions = {
     numTestsKeptInMemory: 0,
 
     e2e: {
-        specPattern: `cypress/tests/**/*.cy.ts`,
+        specPattern: `cypress/tests/desktop/**/*.cy.ts`,
```
Comment (splincode) [inline projects/demo-integrations/cypress.config.ts:47]: cypress.config.ts extends cypress.config.desktop.ts

File: projects/demo-integrations/cypress/tests/desktop/addon-doc/navigation.cy.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import {WAIT_BEFORE_SCREENSHOT} from '../../support/shared.entities';
+import {WAIT_BEFORE_SCREENSHOT} from '../../../support/shared.entities';
```
Comment (splincode) [inline projects/demo-integrations/cypress/tests/desktop/addon-doc/navigation.cy.ts:1]: `import {WAIT_BEFORE_SCREENSHOT} from '@demo-integrations/support/shared.entities';`

add new `path` to `paths` properties in tsconfig.json

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**